packages/gasket-plugin-mocha/index.js

```diff
@@ -32,8 +36,7 @@ module.exports = {
 
         if (isReactProject) {
           files.add(
-            path.join(__dirname, 'generator', '*'),
-            path.join(__dirname, 'generator', '**', '*')
+            path.join(__dirname, 'generator', 'test', 'mocha-watch-cleanup-after-each.js')
```

Comment:
Is there supposed to be a file for this included with the PR?

Reply:
Oh, actually this file was already there. 👍🏻

Reply:
Ah from https://github.com/godaddy/gasket/pull/324 👌🏻